dev.env

```diff
@@ -1,55 +1,45 @@
-PRINT_ENV=dev
-RANCHER_LABEL=dev
-IMAGE_TAG=staging
-NGINX_PORT=8009
-TOMCAT_PORT=8011
-API_URL=//mf-chsdi3.dev.bgdi.ch
-TOMCAT_SERVER_URL=//vpc-alb-bgdi-dev.intra.bgdi.ch
-PRINT_SERVER_HOST=service-print.dev.bgdi.ch
-PRINT_TEMP_DIR=/var/local/print
-PRINT_LOGLEVEL=10
-DEBUG=True
-APACHE_ENTRY_PATH=
-APACHE_BASE_PATH=main
-BRANCH_STAGING=
-GIT_BRANCH=
-MODWSGI_USER=www-data
-INSTALLDIR=/var/www/vhosts/mf-chsdi3/private/chsdi
-WSGI_APP=/var/www/vhosts/mf-chsdi3/private/chsdi/apache/application.wsgi
-KML_TEMP_DIR=/var/local/print/kml
-DEPLOY_TARGET=prod
-MODWSGI_CONFIG=production.ini
 APACHE_BASE_PATH=main
 APACHE_ENTRY_PATH=
 APACHE_LOG_LEVEL=debug
 APACHE_PORT=8009
-API_URL=//api3.geo.admin.ch
+API_URL=//sys-api3.dev.bgdi.ch
 AWS_DEFAULT_REGION=eu-central-1
+BRANCH_STAGING=
 CACHE_CONTROL=no-cache
-DATAGEOADMINHOST=data.geo.admin.ch
+DATAGEOADMINHOST=sys-data.dev.bgdi.ch
 DBHOST=pg.bgdi.ch
 DBPORT=5432
 DBSTAGING=dev
+DEBUG=True
 DEPLOY_TARGET=dev
 DYNAMIC_TRANSLATION=1
-GEOADMINHOST=map.geo.admin.ch
-GEODATA_STAGING=dev
-HOST=api3.geo.admin.ch
+GEOADMINHOST=mf-geoadmin3.dev.bgdi.ch
+GEODATA_STAGING=test
+GIT_BRANCH=
+HOST=sys-api3.dev.bgdi.ch
 HTTP_PROXY=http://ec2-52-28-118-239.eu-central-1.compute.amazonaws.com:80
+IMAGE_TAG=staging
 INSTALLDIR=/var/www/vhosts/mf-chsdi3/private/chsdi
 KEEP_VERSION=true
 KML_TEMP_DIR=/var/local/print/kml
 LINKEDDATAHOST=https://ld.geo.admin.ch
 MODWSGI_CONFIG=development.ini
```

Comment:
Not sure if this is correct. Think I also saw `production.ini` for this var somewhere.. Do you know @procrastinatio?

Reply:
Both `production.ini` `development.ini`are tmplate generated. So we may use only one template or have three more static files